pkg/loop/internal/relayerset/server.go

```diff
@@ -50,6 +52,7 @@ type Server struct {
 
 var _ relayerset.RelayerSetServer = (*Server)(nil)
 var _ evmpb.EVMServer = (*Server)(nil)
+var _ tonpb.TONServer = (*Server)(nil)
```

Comment:
As we add `TONServer`, `EVMServer`, etc., backed by the same `relayerset.Server`, we should consider refactoring this to avoid method name collisions. It makes sense to split each protocol handler into its own server implementation to keep things modular and prevent conflicts as the API surface grows.